app/views/api/v2/ansible_override_values/index.json.rabl

```diff
@@ -0,0 +1,3 @@
+collection @override_values
+
+extends 'api/v2/ansible_override_values/show'
```

Comment:
Why don't we have `base` instead, that would extend `override_values/main` to stay consistent?
Not a blocker tho